05/app.js

```diff
@@ -1,53 +1,79 @@
 const stats = {
-    paragraphs: {
-        'p1': 0,
-    },
-    links: {
-        '/dolor.html': 0,
-    }
+  paragraphs: {
+    p1: 0,
+    p2: 0,
+    p3: 0,
+  },
+  links: {
+    '/dolor.html': 0,
+    '/consectetur.html': 0,
+    '/adipisicing-elite.html': 0,
```

Comment:
Spróbuj to zadanie zrobić w taki sposób, aby dodanie kolejnego linku czy paragrafu nie powodowało przymusu dodawania kodu do zmiennej `stats` :)

PS. Wystarczy w funkcjach liczących sprawdzać czy takie element istnieje, jeśli nie to trzeba go dodać z wartością początkową ;)